src-tauri/Cargo.lock

```diff
@@ -78,7 +78,9 @@ dependencies = [
  "async-trait",
```

Comment:
Unsure if we want the `lock` file, maybe change the `toml` file instead (though I don't see new packages inline)? Is this a merge leftover?

Reply:
I did not manually change lock file. This branch carried over changes from the branch where I implemented postgresql, so toml file also included those extra packages. Since we don't use them anymore, I removed them from toml, which triggered the changes in lock (not visible in the last commit, but in earlier ones).